src/main/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsController.java

```diff
@@ -225,4 +232,16 @@ public Mono<ResponseEntity<SessionValidateResponseDto>> validateSession(
                 .map(transactionId -> new SessionValidateResponseDto().transactionId(transactionId))
                 .map(ResponseEntity::ok);
     }
+
+    @Override
+    public Mono<ResponseEntity<Void>> updateSession(
+                                                    String id,
+                                                    String sessionId,
+                                                    Mono<PatchSessionRequestDto> patchSessionRequestDto,
+                                                    ServerWebExchange exchange
+    ) {
+        return patchSessionRequestDto
+                .flatMap(updateData -> paymentMethodService.updateSession(id, sessionId, updateData))
+                .map(ignored -> ResponseEntity.ok(null));
```

Comment:
Shouldn't it return a 204 http status?
```suggestion
                .map(ignored -> ResponseEntity::noContent);
```

Reply:
Fixed in 4a814dd